### PR TITLE
Add postinstall command to crew

### PR DESCRIPTION
**Pull request by uberhacker**
Should help in those instances when the messages scroll past the viewable screen.

### CODE REVIEW

**Comment by JL2210** [comment]
Yay!

File: crew [hunk 1]
```diff
@@ -927,6 +932,17 @@ def list_command (args)
   end
 end
 
+def postinstall_command (args)
+  args["<name>"].each do |name|
+    set_package name, true
+    if @device[:installed_packages].any? do |elem| elem[:name] == name end
+      @pkg.postinstall
+    else
+      puts "Package #{name} is not installed. :(".lightred
```
Comment (JL2210) [inline crew]: I don't think it should require the package to be installed.

Comment (uberhacker) [reply crew]: Actually, it does in several instances where files are expected to exist.  When you think about it, it does make sense.

Comment (JL2210) [reply crew]: Yes, it could also overwrite user configuration. Just output a warning if the package isn't installed.

Comment (uberhacker) [reply crew]: That's what I'm doing.

Comment (JL2210) [reply crew]: Yes, but you should still go through with the command. You know, there's a difference between a warning and an error.

Comment (uberhacker) [reply crew]: No, I shouldn't.  See above.

Comment (JL2210) [reply crew]: All right. Other than that, this is fine.

Comment (JL2210) [reply crew]: Is there any way you could add an option to force it? I've been looking forward to this for a long time.

Comment (uberhacker) [reply crew]: Why would you need to run postinstall on an uninstalled package?

Comment (JL2210) [reply crew]: Testing a new package and making sure you haven't forgotten `.lightblue` without installing it? Making sure there aren't any typos?

Comment (uberhacker) [reply crew]: You wouldn't see that at the end anyway?

Comment (JL2210) [reply crew]: I don't want to go through the whole build process again just to see the postinstall when I've changed it.

Comment (uberhacker) [reply crew]: Wait, testing a package includes installing it which would also output the text at the end.  You really should be testing it with the install anyway.

Comment (JL2210) [reply crew]: I don't want to build a package again when all I've changed is the postinstall.

**Pull request merged**